inc/editor/patterns/letter-from-the-ceo.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Block pattern for a full-wdth group containing a header and two recent posts.
+ */
+
+namespace WMF\Reports\Editor\Patterns\LetterFromTheCeo;
+
+const NAME = 'wmf-reports/letter-from-the-ceo';
+
+const PATTERN = <<<CONTENT
+<!-- wp:group {"templateLock":"all","lock":{"move":true,"remove":true},"className":"wmf-pattern-letter-from-the-ceo","layout":{"type":"constrained"}} -->
+<div class="wp-block-group wmf-pattern-letter-from-the-ceo"><!-- wp:heading {"lock":{"move":true,"remove":true}} -->
+<h2 class="wp-block-heading">Letter from the CEO</h2>
+<!-- /wp:heading -->
+
+<!-- wp:quote -->
+<blockquote class="wp-block-quote"><!-- wp:paragraph {"lock":{"move":false,"remove":false}} -->
+<p>“Wikipedia’s founding principles are as relevant as ever.”</p>
+<!-- /wp:paragraph --></blockquote>
+<!-- /wp:quote -->
+
+<!-- wp:shiro/profile {"profile_id":65978} /-->
```

Comment:
Is this going to cause problems on the Endowment site where we can't use shiro blocks?

Reply:
Good catch, I think it will. @philipjohn We should use a media-text block here